recipes-kernel/linux/files/0001-iot2050-add-iot2050-platform-support.patch

```diff
@@ -797,24 +832,56 @@ index 000000000000..c585fdfcc5cc
 +&eip76d_trng {
 +	status = "disabled";
 +};
-diff --git a/arch/arm64/boot/dts/ti/k3-am6528-iot2050-basic-common.dtsi b/arch/arm64/boot/dts/ti/k3-am6528-iot2050-basic-common.dtsi
+diff --git a/arch/arm64/boot/dts/ti/k3-am6528-iot2050-basic-oldfw.dts b/arch/arm64/boot/dts/ti/k3-am6528-iot2050-basic-oldfw.dts
 new file mode 100644
-index 000000000000..9a2fcf02accf
+index 000000000000..a3271785d93e
 --- /dev/null
-+++ b/arch/arm64/boot/dts/ti/k3-am6528-iot2050-basic-common.dtsi
-@@ -0,0 +1,55 @@
++++ b/arch/arm64/boot/dts/ti/k3-am6528-iot2050-basic-oldfw.dts
+@@ -0,0 +1,10 @@
 +// SPDX-License-Identifier: GPL-2.0
 +/*
-+ * (C) Copyright 2020 Siemens AG
++ * (C) Copyright 2018-2020 Siemens AG
++ */
++
++/dts-v1/;
++
++#include "k3-am65-iot2050.dtsi"
```

Comment:
should k3-am65-iot2050.dtsi be included into basic.dtsi or advanced.dtsi ?

Reply:
seems to me not a good idea to include dtsi in dtsi, better follow my change, one benefit is that the including relationship is clear and straightforward.